package.json

```diff
@@ -41,8 +41,9 @@
     "@typescript-eslint/parser": "^4.28.1",
     "@vuepress/plugin-google-analytics": "^1.8.2",
     "archiver": "^5.3.0",
-    "chart.js": "^3.4.0",
-    "chartjs-test-utils": "^0.3.0",
+    "chart.js": "^3.8.0",
+    "chartjs-chart-matrix": "^1.1.1",
```

Comment:
According #326, it seems that the breaking change has been introduced in the matrix plugin. Wouldn't it make sense to have this test also in the matrix chart repository (I can't find it)?

Reply:
It used to use Rectangle element from Chart.js v2, so I think datalabels plugin thought it was a bar chart and that's why it worked. But yes, a test with datalabels plugin could be added there (I'm quite confident it does not break :D).